packages/paste-core/components/disclosure/src/DisclosureHeading.tsx

```diff
@@ -1,77 +1,114 @@
 import * as React from 'react';
 import {useTheme} from '@twilio-paste/theme';
-import type {BorderRadius} from '@twilio-paste/style-props';
 import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
 import {Heading, HeadingPropTypes} from '@twilio-paste/heading';
-import {ChevronDisclosureExpandedIcon} from '@twilio-paste/icons/esm/ChevronDisclosureExpandedIcon';
+import {ChevronRightIcon} from '@twilio-paste/icons/esm/ChevronRightIcon';
```

Comment:
todo: Can you use ChevronDisclosureIcon here instead, and replace the SVG in ChevronDisclosure with this one?:

![ChevronDisclosure](https://user-images.githubusercontent.com/3821840/214147917-82014980-cc4d-4d95-a43b-8e598e9a92ee.svg)

Can you also deprecate ChevronDisclosureExpandedIcon and ChevronDisclosureCollapsedIcon? Not sure if "deprecate" is the right word here, or if we just delist icons from the website when we don't want people using them anymore. We've done it in the past—I'm just not 100% sure what we do to the icons.

Sorry, these changes were documented in the google doc, but it prob would've been better to talk through them.

Reply:
Oh! sorry I totally missed that. Updated.